spec/javascripts/Users/components/FeatureAccess.spec.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import { FeatureAccess } from 'Users/components/FeatureAccess'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+let wrapper
+
+function getWrapper (testProps) {
```

Comment:
You are not passing `testProps` in the scope of this file

Reply:
Well it's a test template. `getWrapper` is prepared for whenever `testProps` are needed.

Reply:
If this were a method imported from somewhere else, it would have sense. But if you are declaring the same function in every file it doesn't look like a template anymore.